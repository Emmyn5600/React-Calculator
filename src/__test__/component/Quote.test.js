import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/Quote';

describe('Quote', () => {
  it('it should render Quote page', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
