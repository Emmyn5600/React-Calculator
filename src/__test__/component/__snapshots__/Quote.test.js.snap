// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home it should render Quote page 1`] = `
<div>
  <p
    className="qoute"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. –William Paul Thurston
  </p>
</div>
`;

exports[`Home it should render home navbar 1`] = `
<div>
  <p
    className="qoute"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. –William Paul Thurston
  </p>
</div>
`;

exports[`Quote it should render Quote page 1`] = `
<div>
  <p
    className="qoute"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. –William Paul Thurston
  </p>
</div>
`;
